app/src/main/java/com/imdbmovieapp/data/local/mapper/MovieDomainToMovieEntityMapper.kt

```diff
@@ -0,0 +1,20 @@
+package com.imdbmovieapp.data.local.mapper
+
+import com.imdbmovieapp.data.local.entity.FavoriteMovieEntity
+import com.imdbmovieapp.domain.model.FavoriteMovieDomain
+import com.imdbmovieapp.utils.BaseMapper
+
+class MovieDomainToMovieEntityMapper : BaseMapper<FavoriteMovieDomain, FavoriteMovieEntity> {
+    override fun mapModel(model: FavoriteMovieDomain): FavoriteMovieEntity {
+        return with(model) {
+            FavoriteMovieEntity(
+                id = id,
+                title = title,
+                year = year,
+                genre = genre,
+                image = image,
+                isFavorite = isFavorite
+            )
+        }
+    }
```

Comment:
You also can write like this 
 `   override fun mapModel(model: FavoriteMovieDomain) = with(model) {
        FavoriteMovieEntity(
            id = id,
            title = title,
            year = year,
            genre = genre,
            image = image,
            isFavorite = isFavorite
        )
    }`
    
    Matter of taste, but it looks smoother for me